app/models/school_data_importer.rb

```diff
@@ -2,6 +2,12 @@
 require "csv"
 
 class SchoolDataImporter
+  # Returns the URL for today's schools data CSV file
+  def self.gias_schools_csv_url
```

Comment:
What's the rationale behind making this a class method rather than an instance method?

Reply:
Mostly because it makes the specs for this class simpler – we can stub the requests with this value without having to instantiate an instance, or replicate the implementation to get the correct time-stamped URL.

Reply:
Ah, yeah, of course. I think that became clearer as my review progressed. Good thinking 👍

Reply:
I mean it's not a great reason to be fair... Might have made more sense to keep it private and freeze time in the specs...

Reply:
OK, I've changed my mind and pushed up an additional commit to make this method private again.